app/views/proposals/details/_attachment.html.haml

```diff
@@ -16,3 +14,5 @@
           .medium-12.column
             = render partial: "proposals/details/attachment_list",
               locals: { proposal: @proposal }
+            - if !@proposal.completed? && !@proposal.canceled?
```

Comment:
I think @nickbristow just made a change to this line. Going to follow up to make sure you two dont overwrite each other.